### PR TITLE
Adding new army json

### PR DESCRIPTION
**Pull request by mpolakow**
Added new army - Outpost.json the last human army 
Modified json schema to include new abilities and modifiers

### CODE REVIEW

File: src/resources/armies/army.schema.json [hunk 1]
```diff
@@ -159,7 +159,8 @@
                     "mobility",
                     "pushback",
                     "small_bomb",
-                    "venom"
+                    "venom",
+		    "sniper"
```
Comment (tom3q) [inline src/resources/armies/army.schema.json]: Please keep them sorted alphabetically.

File: src/resources/armies/army.schema.json [hunk 2]
```diff
@@ -185,7 +186,9 @@
                     "melee",
                     "net",
                     "quatermaster",
-                    "riffle"
+                    "riffle",
+		    "scoper",
+		    "recon"
```
Comment (tom3q) [inline src/resources/armies/army.schema.json]: Ditto.

**Review by tom3q — CHANGES_REQUESTED** [review]

File: src/resources/armies/army.schema.json (hunk 1 above)
Comment (tom3q) [reply src/resources/armies/army.schema.json]: Uh, and fix the tabs again...

**Comment by Nucleoprotein** [comment]
Maybe use .editorconfig? ( https://editorconfig.org/ )

**Comment by tom3q** [comment]
> Maybe use .editorconfig? ( https://editorconfig.org/ )

Sounds like a good idea. I'll check it in the evening.

File: src/resources/armies/army.schema.json (hunk 1 above)
Comment (mpolakow) [reply src/resources/armies/army.schema.json]: Fixed

File: src/resources/armies/army.schema.json (hunk 2 above)
Comment (mpolakow) [reply src/resources/armies/army.schema.json]: Fixed

**Comment by tom3q** [comment]
Please rebase.

**Pull request merged**